gateway/1.4/modules/ROOT/pages/index.adoc

```diff
@@ -98,8 +98,9 @@ You can install Flex Gateway in a variety of ways:
 
 * A sidecar to a backend application in a Docker container, thereby protecting a single backend application
 
-* An Ingress controller in Kubernetes, for high-availability, high-performance use cases
+* A Kubernetes cluster for high-availability, high-performance use cases
 
+//TODO: "ingress controller" ok?
```

Comment:
@seblaz What do you think we should say below. See "Otherwise, use an Ingress controller in Kubernetes."

Reply:
done